__tests__/app.test.js

```diff
@@ -145,15 +144,37 @@ describe("/api/articles/:article_id/comments", () => {
       .get("/api/articles/lizzie/comments")
       .expect(400)
       .then(({ body }) => {
-        expect(body.msg).toBe("Sorry, try again!")
+        expect(body.msg).toBe("Sorry - not found!")
       })
     })
     test("404: throws error when article id is correct type but not on the table", () => {
       return request(app)
       .get("/api/articles/500/comments")
       .expect(404)
       .then(({ body }) => {
-        expect(body.err.msg).toBe("Sorry, I couldn't find that!")
+        expect(body.msg).toBe("Sorry, I couldn't find that article!")
       })
     })
-  })
\ No newline at end of file
+  })
+  describe("/api/articles/:article_id/comments POST", () => {
+    test("201: successfully inserts a comment", () => {
+      return request(app)
+      .post("/api/articles/2/comments")
+      .expect(201)
+      .send({username: "lurker", body: "I just got engaged!!!"})
+      .then(({ body }) => {
+        expect(body.msg).toBe("comment posted!")
+        expect(body.comment).toBe("I just got engaged!!!")
+      })
```

Comment:
We should be expecting body to have a key of comment which holds the actual comment data here. 

We also want to be expecting the whole comment object that was posted back from the database not just the body, and msg does not need to be included for this.